util/logger.js

```diff
@@ -115,10 +102,8 @@ function createDirectory(category) {
  * @returns {string} The created filename
  */
 function createFilename(category, index) {
-	let filename = category + "_" + index + ".log";
-	if (config.log.compression.active) {
-		filename += ".gz";
-	}
+	let filename = `${category}_${index}.log"`;
```

Comment:
```suggestion
	let filename = `${category}_${index}.log`;
```
Fix filenames ending in `"`